hapi/hedera-protobufs/services/consensus_update_topic.proto

```diff
@@ -87,4 +88,37 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <p>
+     * If unset in state, this field MUST NOT be set.<br/>
+     * If unset in this transaction and set in state, the current value SHALL
+     *  be removed.<br/>
+     * If set in this transaction, the existing key MUST sign this transaction.
+     */
+    Key fee_schedule_key = 10;
+
+    /**
+     * A wrapper of a set of keys that are allowed to submit messages to the topic without
+     * paying the topic's custom fees.
+     * <p>
+     * If the wrapper is not set, the current set of keys is unchanged.<br/>
+     * If the wrapper is set, but contains an empty list, the list will be updated to an empty list.
+     */
+    FeeExemptKeyList fee_exempt_key_list = 11;
+
+    /**
+     * A wrapper of a set of custom fee definitions.<br/>
+     * These are fees to be assessed for each submit to this topic.
+     * <p>
+     * If the wrapper is not set, the current set of keys is unchanged.<br/>
+     * If the wrapper is set, but contains an empty list, the list will be updated to an empty list.
```

Comment:
Copy/paste error, and incorrect keyword usage:
```suggestion
     * If this field is not set, the current set of custom fees
     * SHALL NOT change.<br/>
     * If this field is set, but contains an empty list, all current custom fees
     * SHALL be removed.
```

Reply:
Fixed